Source/AliveLibAE/MainMenu.cpp

```diff
@@ -1569,39 +1622,39 @@ void MainMenuController::AbeMotions_Render_4D25E0(int ** ot)
     RenderOnScreenTextHelper(ot, &sAbeMotions_562448[0], ALIVE_COUNTOF(sAbeMotions_562448), 0);
 }
 
-signed int MainMenuController::Gamespeak_Update_4D1FC0(DWORD input_held)
+MainMenuNextCam MainMenuController::Gamespeak_Update_4D1FC0(DWORD input_held)
 {
     field_230_target_entry_index = 0;
 
     if (input_held & InputCommands::eBack)
     {
-        return MainMenuCams::eMainMenuCam;
+        return { MainMenuCams::eMainMenuCam };
     }
 
     if (!(input_held & InputCommands::eUnPause_OrConfirm))
     {
-        return MainMenuCams::eNoChange;
+        return { MainMenuCams::eNoChange };
     }
 
     switch (field_1FC_button_index)
     {
     case 0:
-        return 0xFFFF000E;
+        return { MainMenuCams::eAbeGamespeakCam, NO_SELECTABLE_BUTTONS };
         break;
```

Comment:
not sure what was going on here, the breaks are pointless because of the unconditional returns ,might as well nuke them